experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -679,17 +696,17 @@ const ChartTile: React.FC<{
           subscribe={channel}
           header={`${tile.title}*`}
           variable={tileStatVars.join(" ")}
-          parentPlace="Earth"
-          childPlaceType="Country"
+          parentPlace={placeDcid}
+          childPlaceType={childPlaceType}
           showExploreMore={true}
```

Comment:
For now, what do you think of not displaying map unless `childPlaceType` is "Country" in 689 condition?

It is a one-line fix for the backend data issue?